src/test/java/net/masterthought/cucumber/generators/integrations/FailuresOverviewPageIntegrationTest.java

```diff
@@ -69,21 +69,4 @@ public void generatePage_onJsonWithoutFailedSteps_generatesProperMessage() {
         SummaryAssertion summary = document.getReport();
         assertThat(summary.getEmptyReportMessage()).isEqualTo("You have no failed scenarios in your Cucumber report");
     }
```

Comment:
But then we have no test that validates if data is escaped properly (which is not global. default configuration). Worth to have test for this, even one.